server/test/utils.ts

```diff
@@ -77,4 +75,6 @@ export {
   randomEmptyChance,
   createUser,
   getExpectedUserObject,
+  makeUserObject,
+  createUsers,
```

Comment:
nit: I typically sort exports 1 of 2 ways. Alphabetically, or in the order in which they appear above. I tend to prefer alphabetically.

Reply:
i feel like we can also just export the functions directly, e.g. `export function getExpectedUserObject(user: User) {`

Reply:
Then they'll have to be below all non-exported functions or we'll need to remove one of the ESLint rules I added.